conf/script/build-ha-update

```diff
@@ -89,7 +89,14 @@ if [[ -f $consul_kv_dump ]]; then
 fi
 $consul_bin/consul kv export > $consul_kv_dump
 
+sudo pcs cluster standby --all
+
 reset_all
+
+sudo pcs cluster unstandby --all
+sleep 2
```

Comment:
Why we need sleep here? Is 2 seconds sufficient?

Reply:
Added delay to give some time to resources to start successfully before doing the resource cleanup. I tested and went well. So, I feel 2 secs delay is sufficient.

Reply:
We should be careful with adding delays in the software update path. Already we had tasks to optimize the software update time. I would recommend removing this delay if it is not required absolutely.

Reply:
Sure. I will give a try without delay and update the results.

Reply:
It worked fine without delay also. So, removing delay in the next patch as it's not required.